src/main/serverHelper.ts

```diff
@@ -0,0 +1,71 @@
+import http from 'http';
+import Logging from '@/utils/logging';
+
+const console = Logging.server;
+
+export function basicAuth(expectedUser: string, expectedPassword: string, authString: string): boolean {
+  if (!authString) {
+    console.log('Auth failure: no username+password given');
+
+    return false;
+  }
+  const m = /^Basic\s+(.*)/.exec(authString);
```

Comment:
```suggestion
  const m = /^Basic\s+(.*)/i.exec(authString);
```

Reply:
I always thought this is case-sensitive, but looking at RFC 2617 I see that it isn't (while the actual token value obviously is)